base/views.py

```diff
@@ -20,25 +20,6 @@
 from units.utils import get_default_unit
 
 
-def breadcrumbs(request):
```

Comment:
Nice!

Reply:
Yeah, it's weird that that code wasn't being used except by that view which also wasn't being used. I didn't trust it at first but that's what I determined. By all means if you understand differently, let me know!